### DISCUSSION TITLE
"Full Combo" on lazer has a different max combo than stable

### DISCUSSION THREAD

**Discussion by Cesan** [ppy/osu#15179] [Q&A] [answered]
Hi, I just played a map in lazer and got an FC (346x) but the max combo is way lower than actually possible (515x).
When viewing the score in stable it shows the correct (full) combo value while viewing but shows the false lazer value at the score screen. 
If this is normal and a wanted behaviour, I didn't know that and I'm sorry for wasting time.
Thanks, I attached the score/replay file of the play below.
[Replay as zip file](https://github.com/ppy/osu/files/7364725/hoki.on.osu.playing.Okazaki.Taiiku.-.Kimi.no.Bouken.TV.Size.Sotarks.NiNo.s.Insane.zip)

**Comment by bdach** [comment, marked as answer]
Slider ends do not contribute to combo in lazer, contrary to stable. This is likely what you're seeing.

**Reply by Cesan** [reply]
Alright, I think that's why. It's just that the pp values displayed ingame in lazer are not inline with what should be calculated with the slider ends considered. Is that normal? The score above gives me around 87 in lazer but in stable it would give me around 113 or so.

**Reply by integerrr** [reply]
pp values will likely be different to ones shown in stable

**Reply by bdach** [reply]
> pp values will likely be different to ones shown in stable

...due to them being also derived from max combo.

**Reply by Cesan** [reply]
Alright, thanks for your time. Really appreciate it.
